scripts/globals/cutscenes.lua

```diff
@@ -0,0 +1,7 @@
+xi = xi or {}
+xi.cutscenes = {}
+
+xi.cutscenes.params =
+{
+    NO_OTHER_ENTITY = 0x93 -- Only Displays Relative Player and NPCs to that Cutscene
```

Comment:
I'm no expert in this kind of stuff, but, I would guess 0x93 is a combination of flags.
concretely

0x01
0x02
0x10
0x80

Each of which would do something different and combined would result in that value (0x93)

Reply:
Would need comment from Velyn there, she provided me the original bitflag specific to the cutscenes I did as exmaples.

Reply:
She got the optional params in captures, so we just need to start digging and fill in the blank from captures.

Reply:
Or pick an obvious cutscene and trying random bits to see what each does as @jmcmorris suggests. For the initial commit I can add comments about others existing. Then add more as I have time to check and fix other cutscenes. (I have a few others on my list coming up)

Reply:
yeah lets get the skeleton of how to try it out in. I saw it was made backward compatible to continue using cs as is without having to go update every cs line in lua - good call. I 0x12 should hit the 2 bits to perform what the comment is, no idea what 0x01 and 0x80 are yet tho. we can figure it out later and revise the enumeration. 

Not sure how we want to tackle representing a combined flag value in human readable form after that tho, maybe some kind of short lua helper to avoid doing magic numbers and avoid doing inline flag+flag+flag

Reply:
For common flag combinations we can set those up in the `xi.cutscene.flags` to avoid having to inline flag+flag+flag

Reply:
> For common flag combinations we can set those up in the `xi.cutscene.flags` to avoid having to inline flag+flag+flag

I kinda disleek doing that but if its the best option we'll do it. It's happening to some of the flags on char entities already (ones currently not properly enum'd, which I am tackling - there's just no better way unfortunately)

Reply:
Just let me know what else is specifically desired for this initial commit

Reply:
we will, but I make no promises that we won't be chatty while we think about it!

:smiley:

Reply:
No problem, its late for me. So i'll look at comments in the morning.